src/main/webapp/WEB-INF/views/drivers/add.jsp

```diff
@@ -23,6 +26,12 @@
         <td>
             <input type="text" name="license_number" form="driver" required>
         </td>
+        <td>
+            <input type="text" name="login" form="driver" required>
+        </td>
+        <td>
+            <input type="text" name="password" form="driver" required>
```

Comment:
type should be `password`
please, provide a screenshot with this page after you enter login and password

Reply:
Yeah, It has worked only after I add @Service annotation

![add](https://user-images.githubusercontent.com/78808019/204595078-09c20f32-2cd0-4b25-b303-15741a0858cc.png)